meetings/2019-11/nov-18.md

```diff
@@ -0,0 +1,34 @@
+# Meeting Notes - 2019-11-18
+
+## Attendees
+
+- [Alan Dong](https://github.com/adong)
+- [Keisuke Kumada](https://github.com/kumada626)
+
+## Agenda
+
+From VMG:
+
+- [Force Push](https://github.com/screwdriver-cd/screwdriver/issues/1675) update during [freezeWindow](https://docs.screwdriver.cd/user-guide/configuration/workflow.html) time
+- [Disk Optimzation](https://github.com/screwdriver-cd/screwdriver/issues/1830)
+- [Parallel Fork](https://github.com/screwdriver-cd/screwdriver/issues/1710)
+
+From YJapan:
+
+- [SD Local](https://github.com/screwdriver-cd/screwdriver/issues/1314)
+- [Template/Command list screen shows wrong VERSION](https://github.com/screwdriver-cd/screwdriver/issues/1723)
+- [Performance issue of webhook](https://github.com/screwdriver-cd/screwdriver/issues/1468)
+- [feature request: Teardown Job](https://github.com/screwdriver-cd/screwdriver/issues/1834)
```

Comment:
We also added a [feature request: Pipeline token should be able to create Secrets](https://github.com/screwdriver-cd/screwdriver/issues/1847)

Reply:
Updated